src/command_modules/azure-cli-kusto/azure/cli/command_modules/kusto/_params.py

```diff
@@ -0,0 +1,36 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+from knack.arguments import CLIArgumentType
+
+from azure.cli.core.commands.parameters import (name_type)
+from azure.mgmt.kusto.models import AzureSkuName
+from azure.cli.core.commands.parameters import (get_enum_type)
+
+
+def load_arguments(self, _):
+
+    # Kusto clusters
+    sku_arg_type = CLIArgumentType(help='The name of the sku.',
+                                   arg_type=get_enum_type(AzureSkuName))
+
+    with self.argument_context('kusto cluster') as c:
+        c.ignore('kusto_management_request_options')
+        c.argument('cluster_name', arg_type=name_type, help='The name of the cluster.', id_part='name')
+        c.argument('sku', arg_type=sku_arg_type)
```

Comment:
You also need `sku_name` to reference this arg_type so that it will be correctly applied to your `cluster update` command.